README.md

```diff
@@ -91,6 +91,30 @@ handles a multitude of input configuration types:
 In the absence of this flag, a default configuration is loaded from `.semgrep.yml`
 or multiple files matching `.semgrep/**/*.yml`.
 
+### Registry
+
+As mentioned above, you may also specify a `registry_name` as configuration.
+[r2c](https://r2c.dev) provides a [registry](https://github.com/returntocorp/semgrep-rules)
+of configuration files. These rules have been tuned on thousands of repositories
+using our [analysis platform](https://app.r2c.dev).
+
+You can browse the registry at https://semgrep.live/r. To run a set of rules, use a rule ID or namespace.
+
+```bash
+# Run a specific rule
+semgrep --config=https://semgrep.live/c/r/java.spring.security.audit.cookie-missing-samesite
+
+# Run a set of rules
+semgrep --config=https://semgrep.live/c/r/java.spring.security
+```
+
+The registry features rules for many programming errors, including security issues and correctness bugs. Security rules are annotated with CWE and OWASP metadata when applicable. OWASP rule coverage per language is displayed below.
+
+<p align="center">
+    <img width="600" src="https://web-assets.r2c.dev/semgrep-rules-owasp-coverage-20200520.png" style="max-width:100%;" />
```

Comment:
I was thinking we'd move this up even higher. Now that we're starting to hook people on the idea of `semgrep` it'd be great to start strongly advocating for using the registry. I think AppSec-types will love this kind of coverage.

My original thoughts were to include the bar chart in the "Overview" right under the example patterns. We can then add something like "For more information see <link-to-registry> or the registry section below."

Thoughts? @ievans may have thoughts here too.

Reply:
I wouldn't put this chart here in the README -- seems much more useful in the enterprise deck, or as something automatically generated on semgrep.live.

Reply:
We can pretty much put this chart wherever we want. I added an endpoint to calculate the CWE and OWASP coverage to semgrep.live yesterday. Example: https://semgrep.live/api/registry/r2c.python.django.security.injection/stats

(This chart isn't auto-generated yet, but you can see how it would be easy to do.)

I agree that these charts should go on semgrep.live too; but we can also include them here if we think it's flashy enough.

An alternative is making the chart less obtrusive and making it more prominent in the semgrep-rules README. lmk your thoughts @mschwager @ievans

Reply:
> I wouldn't put this chart here in the README -- seems much more useful in the enterprise deck, or as something automatically generated on semgrep.live.

Why not both? My thoughts here were that a prominent display of these stats would entice users to check out and use the registry rules. Which makes me wonder, are we currently gathering any stats on which registry rules are being used most? I know when I'm skimming a README it's easy enough to pass over text, but a big image with relevant information is harder to miss. The downside I see is that readers may think we _only_ do OWASP Top 10-type stuff.

Overall, I don't have too strong of feelings, but I agree that we should be including this data wherever is relevant - I think it's a great push for the registry.